Backbone.API/Configuration/BackboneConfiguration.cs

```diff
@@ -40,7 +40,7 @@ public class CorsConfiguration
     public class SwaggerUiConfiguration
     {
         [Required]
-        public string TokenUrl { get; set; }
+        public string TokenUrl { get; set; } = "";
```

Comment:
can we use string.Empty here?

Reply:
I wanted to address changing "" to string.Empty in a separate PR.